### PR TITLE
Log4j v8

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Updated playbook with the details of the newly released vulnerability for Log4j.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md [hunk 1]
```diff
@@ -6,34 +6,33 @@ On Dec. 14 2021, another vulnerability was discovered related the log4j 0-day ex
...
+
 **The playbook includes the following tasks:**
 
-* Collecting Indicators - related known indicators from several sources.
-* Investigation and Hunting - indicators and exploitation patterns hunting using PAN-OS, Cortex XDR and SIEM products.
-* Search for potentially vulnerable servers using Xpanse and Prisma Cloud.
-* Remediation - Blocking indicators automatically or manually based on **playbook inputs**.
-* Mitigations:
-    * Apache official CVE-2021-44228, CVE-2021-45046 patch.
-    * Unit42 recommended mitigations.
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md:21]: Use this description

Comment (altmannyarden) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md:21]: that was removed

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 2]
```diff
@@ -3522,248 +3535,8 @@ tasks:
...
-    isoversize: false
-    isautoswitchedtoquietmode: false
-  "92":
-    id: "92"
-    taskid: 5ed45bdc-97d9-423f-8d1e-f1d1d24c23e3
-    type: title
-    task:
-      id: 5ed45bdc-97d9-423f-8d1e-f1d1d24c23e3
-      version: -1
-      name: Prisma Cloud
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:3543]: Maybe you are not using the latest playbook version? why was this removed?

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 3]
```diff
@@ -10,17 +12,23 @@ description: "Critical RCE Vulnerability: log4j - CVE-2021-44228\n\nOn Dec. 9, 2
...
+  \ Appender with a data source referencing a JNDI URI which can execute remote code.\
+  \ This issue is fixed by limiting JNDI data source names to the java protocol in\
+  \ Log4j2 versions 2.17.1, 2.12.4, and 2.3.2.\n\n**Affected Version**\n\nApache Log4j\
+  \ 2.x <= 2.15.0-rc1\n\nThis playbook should be triggered manually or can be configured\
+  \ as a job.\nPlease create a new incident and choose the **CVE-2021-44228 - Log4j\
+  \ RCE** playbook and **Rapid Breach Response** incident type.\n\n**The playbook\
+  \ includes the following tasks:**\n\n* Collect related known indicators from several\
+  \ sources.\n* Indicators and exploitation patterns hunting using PAN-OS, Cortex\
+  \ XDR and SIEM products.\n*Search for possible vulnerable servers using Xpanse.\n\
+  * Block indicators automatically or manually.\n\n**Mitigations:**\n* Apache official\
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:28]: Search for possible vulnerable servers using Xpanse and **Prisma Cloud.**
where is prisma part? :)

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 4]
```diff
@@ -3790,19 +3564,15 @@ view: |-
...
     "paper": {
       "dimensions": {
-        "height": 6055,
-        "width": 5900,
-        "x": -1670,
-        "y": -1620
+        "height": 5725,
+        "width": 4810,
+        "x": -1760,
+        "y": -1290
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:3575]: Need to add the new CVE in the playbook inputs.

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 5]
```diff
@@ -3877,7 +3896,7 @@ inputs:
   playbookInputQuery:
 - key: BlockIndicatorsAutomatically
   value:
-    simple: "False"
+    simple: "True"
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:3899]: should add the new cve to the playbook input "RelatedCVEs"

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 6]
```diff
@@ -3042,7 +3060,7 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 670,
+          "x": 720,
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:3063]: Please also add in the prisma tasks - the "skip is automation... unavailable "

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml (hunk 5 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:3899]: Done

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml (hunk 6 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:3063]: Done

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**